tests/integrationtest/r/planner/core/integration_partition.result

```diff
@@ -819,33 +835,33 @@ select * from t where a = 2 and b = "2022-01-02" and c < "Hi";
 a	b	c
 select * from t where a < 2;
 a	b	c
--2147483648	0000-00-00 00:00:00	
 -2147483648	0000-00-00 00:00:00	NULL
```

Comment:
It seems that this result is unstable no matter with / without `--sorted_result`. Maybe it's a bug of `mysql-tester`? Let me take a look 🤔 .

Reply:
https://github.com/pingcap/mysql-tester/pull/108/